common/testing/infologtest.lua

```diff
@@ -0,0 +1,58 @@
+-- 'hidden' test checking infolog.txt for errors, used by headless runs.
+
+local maxErrors = 10
+
+function skip()
+	-- TODO: re-enable. disabled 2025-10-01 in order to get CICD working
+	return true
+end
+
+
+local function skipErrors(line)
+	if string.find(line, 'Could not finalize projectile-texture atlas', nil, true) then
+		return true
+	end
+	if string.find(line, 'Could not finalize Decals', nil, true) then
+		return true
+	end
+	if string.find(line, 'Could not finalize groundFX texture', nil, true) then
+		return true
+	end
+	-- Errors for engine >= 2025.03.X deprecations, remove these
+	-- at a later date when they're removed from BAR too.
+	if string.find(line, '"AnimationMT" is read-only', nil, true) then
+		return true
+	end
```

Comment:
this test seems skipped already so it's probably good not to castrate it. bar is on 2025.04.x anyway afaik